spring-integration-sftp/src/test/java/org/springframework/integration/sftp/dsl/SftpTests.java

```diff
@@ -146,6 +146,33 @@ public void testSftpOutboundFlow() {
 		registration.destroy();
 	}
 
+
+	@Test
+	public void testSftpOutboundFlowWithChmod() {
```

Comment:
I `@Ignore` this test because it doesn't work as expected on Windows: the `rw` permissions are assigned for all principals 😢

Reply:
Yes, I got the same error on windows :( looks like it works only on Linux